src/Server.jl

```diff
@@ -382,12 +382,32 @@ function get_sha_from_branch(reponame, brn; auth = GitHub.AnonymousAuth())
     return nothing, nothing
 end
 
+function is_owned_by_organization(event)
+  return event.repository.owner.typ == "Organization"
+end
+
 function is_comment_by_collaborator(event)
     @debug("Checking if comment is by collaborator")
     user = get_user_login(event.payload)
     return iscollaborator(event.repository, user; auth=get_access_token(event))
 end
 
+function is_comment_by_org_owner_or_member(event)
+    @debug("Checking if comment is by repository parent organization owner or member")
+    org = event.repository.owner.login
+    user = get_user_login(event.payload)
+    userorgs = orgs(user)
```

Comment:
```
julia> orgs("nkottary")
(Owner[Owner("JuliaDatabases")], Dict{String,String}())

```

I am an owner/member in 3 other orgs, this just returns one. I tried it with an `auth` param too. Didn't help.